### ISSUE TITLE
Add ability to "subscribe" to calendar

### ISSUE THREAD

**Issue by zlehnherr1** [JICA98/DailyAL#32]
It would be awesome to make the personalized calendars able to be "subscribed" to, allowing the import to other calendars such as google, Samsung, or proton calendar. That way the calendar is in one spot with everything else, and gets updated dynamicall. Not sure of logistics, bur this could allow iOS users the ability of the calendar as well. If this is too difficult, maybe a widget for the home screen.

**Comment by JICA98** [comment]
I have added an option to subsribe which will basically add an event to the google calendar in the latest update. Please check it out - https://github.com/JICA98/DailyAL/releases/tag/2025.8.1%2B100

**Comment by zlehnherr1** [comment]
> I have added an option to subsribe which will basically add an event to the google calendar in the latest update. Please check it out - https://github.com/JICA98/DailyAL/releases/tag/2025.8.1%2B100

Unfortunately that doesn't work for me. I use Proton calendar, and perhaps it's a limitation of that and not DailyAL. It opens in the default system calendar app and not Proton, so I have no way of automatically adding it.

**Comment by JICA98** [comment]
Yeah, in which case, probably I will also have to add an option to download an ".ics" file which can be imported into any calendar. Let me try implementing that.

I will keep this issue open till then. Thanks :)

**Comment by JICA98** [comment]
I have added .ics support with the latest update. Feel free to check it out and let me know if you are able to import it in your proton calendar.

https://github.com/JICA98/DailyAL/releases/tag/2026.2.4%2B106

**Comment by zlehnherr1** [comment]
> I have added .ics support with the latest update. Feel free to check it out and let me know if you are able to import it in your proton calendar.
> 
> https://github.com/JICA98/DailyAL/releases/tag/2026.2.4%2B106

Still can't import the file unfortunately, but it seems to be a limitation of Proton Calendar at this point and not DailyAL. It seems Proton Calendar doesn't support importing an ics with multiple events.
